gateway/requirements.txt

```diff
@@ -1,19 +1,19 @@
 Click==7.0
-Flask==1.0.2
+Flask==1.1.1
 eventlet==0.24.1
 gunicorn==19.9.0
 Flask-Cors==3.0.7
 Flask-Executor==0.9.3
-Flask-SocketIO==3.3.2
+Flask-SocketIO==4.2.1
```

Comment:
Could you try to remove this dependency? I believe that we no longer need it (we're using pure WebSockets to perform the communication instead of SocketIO protocol). It is probably a legacy code.

Reply:
Sure. Thanks